templates/ralus.cfg.erb

```diff
@@ -9,8 +9,8 @@ Software\Symantec\Backup Exec For Windows\Backup Exec\Engine\Agents\Advertise No
 Software\Symantec\Backup Exec For Windows\Backup Exec\Engine\Agents\Advertisement Purge=0
 Software\Symantec\Backup Exec For Windows\Backup Exec\Engine\Agents\Advertising Disabled=0
 Software\Symantec\Backup Exec For Windows\Backup Exec\Engine\Agents\Advertising Interval Minutes=240
-Software\Symantec\Backup Exec For Windows\Backup Exec\Engine\Agents\Agent Directory List_1=<%= scope.lookupvar("::hostname") %>.bak.epost-dev.de
-Software\Symantec\Backup Exec For Windows\Backup Exec\Engine\Agents\Agent Directory List_2=EPD-BE2012
+Software\Symantec\Backup Exec For Windows\Backup Exec\Engine\Agents\Agent Directory List_1=fulitvm016.ad1.proemion.com #make this variable
+Software\Symantec\Backup Exec For Windows\Backup Exec\Engine\Agents\Agent Directory List_2=FULITVM016 #make this variable
```

Comment:
We might want to use variables here instead hardcoded hostnames?

Reply:
@SISheogorath yes that's the final plan, for the first draft i'd like to try with this